src/pages/en/guides/fonts.md

```diff
@@ -0,0 +1,102 @@
+
+## Using a local font file
+
+If you want to add font files to your project, we recommend adding them to your [`public/` directory](http://localhost:3000/en/core-concepts/project-structure/#public). In your CSS you can then register fonts with an [`@font-face` statement](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) and use the `font-family` property to style your site.
+
+### Example
+
+Let’s imagine you have a `DistantGalaxy.woff` font file.
+
+1. Add your font file to `public/fonts/`.
+
+2. Add an `@font-face` statement to your CSS. This could be in a global `.css` file you import or in a `<style>` block in the layout or component where you want to use this font.
+
+    ```css
+    /* Register our custom font family and tell the browser where to find it. */
+    @font-face {
+      font-family: 'DistantGalaxy';
+      src: url('/fonts/DistantGalaxy.woff') format('woff');
+      font-weight: normal;
+      font-style: normal;
+      font-display: swap;
+    }
+    ```
+
+    :::note
+    We don’t include `public` in the font’s source URL because all files in `public` get added to your site’s root directory.
+    :::
+
+3. Use the `font-family` from the `@font-face` statement to style elements in your component or layout. In this example, the `<h1>` heading will have the custom font applied, while the paragraph `<p>` will not.
+
+    ```astro
+    ---
+    ---
+
+    <h1>In a galaxy far, far away...</h1>
+
+    <p>Custom fonts make my headings much cooler!</p>
+
+    <style>
+    h1 {
+      font-family: 'DistantGalaxy';
+    }
+    </style>
+    ```
+
+## Using Fontsource
+
+The [Fontsource](https://fontsource.org/) project simplifies using Google Fonts and other open source fonts. It provides npm modules you can install for the fonts you want to use.
+
+1. Find the font you want to use in [Fontsource’s catalogue](https://fontsource.org/fonts). For this example, we’ll use [Twinkle Star](https://fontsource.org/fonts/twinkle-star).
+
+2. Install the package for your chosen font.
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm i @fontsource/twinkle-star
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm i @fontsource/twinkle-star
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add @fontsource/twinkle-star
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+    :::tip
+    You’ll find the correct package name in the “Quick Installation” section of each font page on Fontsource’s website. It will start with `@fontsource/` followed by the name of the font.
+    :::
+
+3. Import the font package in the layout or component where you want to use the font. Usually you will want to do this in a common layout component to make sure the font is available across your site.
```

Comment:
Nit: Not 100% sure, but I think we are missing a comma here:
```suggestion
3. Import the font package in the layout or component where you want to use the font. Usually, you will want to do this in a common layout component to make sure the font is available across your site.
```

Reply:
I’m not sure either here — a comma is possible but I kind of prefer it without. I’ll wait for @sarah11918 who usually has a good handle on these things.

Reply:
Agree that it could go either way, so it's really personal preference here. And if I were to choose, it *sounds* better without, but I like that a comma would visually stop someone skimming because I think it's actually kind of important to point out the layout thing to make this available site-wide.

Let me throw something else out there, though! Does this rise to the level of "tip"? `<SarahThinksThisIsImportant />`

Reply:
Could be a tip but we already have one a line above, so I’ll go with the comma for now!